solutions/devsprint-leonardo-santos-1/FinanceAppTests/Service/NetworkClientSpy.swift

```diff
@@ -0,0 +1,24 @@
+//
+//  NetworkClientSpy.swift
+//  FinanceAppTests
+//
+//  Created by Leonardo Cunha on 21/06/22.
+//
+
+import Foundation
+@testable import FinanceApp
+
+final class NetworkClientSpy: NetworkClientProtocol {
```

Comment:
Vou deixar o approve no PR e quando o MR do NetworkClientSpy for mergeado, podemos remover esse código e mergear esse PR! Arrasou nos testes!

Reply:
Atualizei a branch, Léo